addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5555,12 +5555,17 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
      * @private
      * @param {boolean} activate toggle the overlay on (true) or off (false)
      */
-    _toggleBgOverlay: function (activate) {
+    _toggleBgOverlay: async function (activate) {
         if (!this.$backgroundOverlay || this.$backgroundOverlay.is('.oe_active') === activate) {
             return;
         }
 
-        if (!activate) {
+        if (activate) {
+            // Make sure element is in visible area
+            this.$target[0].scrollIntoView(false);
```

Comment:
How is it possible the element is not in the viewport? :thinking:

Reply:
Scenario:
- Drop a few blocks
- Select top-most one (or bottom-most)
- Scroll down (or up) until the block is out of the viewport while still being selected
- Click on background positioning button

Reply:
Oh, yes, I am stupid :shrug: However, that part of the bug fix should then be in 14.0 in a dedicated commit :+1:

Reply:
Indeed, that part of the problem has just been extracted to https://github.com/odoo/odoo/pull/75467